cmd/ddev/cmd/config.go

```diff
@@ -368,6 +373,63 @@ func handleMainConfigArgs(cmd *cobra.Command, _ []string, app *ddevapp.DdevApp)
 		}
 	}
 
+	if cmd.Flag("update").Changed {
+		friendlyMsg := fmt.Sprintf("Update flag detected")
+		output.UserOut.Print(friendlyMsg)
+
+		detectedApptype := app.DetectAppType()
+		detectedApptypeMsg := fmt.Sprintf("Detected %s", detectedApptype)
+		output.UserOut.Print(detectedApptypeMsg)
+
+		// For the POC we need ConfigFileOverrideAction to omit app.ConfigExists() check
+		err = app.ConfigFileOverrideAction()
+
+		apptypeMsg := fmt.Sprintf("Apptype is %s", app.Type)
+		output.UserOut.Print(apptypeMsg)
+
+		if app.Type == nodeps.AppTypeDrupal {
```

Comment:
I don't think you should need to mention specific project types here.

Reply:
Yeah, it was only for debugging purposes when I did that. I just thought that we might want to take this chance to change apptype drupal8, drupal9, drupal10 to drupal if we find those.